Packs/Core/Playbooks/playbook-A_mail_forwarding_rule_was_configured_in_Google_Workspace.yml

```diff
@@ -0,0 +1,1784 @@
+                      iscontext: true
+                    right:
+                      value:
+                        simple: "3"
+                accessor: Indicator
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 450
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 65753a7c-d6c1-4592-8094-7a9efe197055
+    type: title
+    task:
+      id: 65753a7c-d6c1-4592-8094-7a9efe197055
+      version: -1
+      name: Verdict
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "37"
+      - "17"
+      - "46"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 440,
+          "y": 1100
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 971290fa-daf4-4510-81d2-610dd2cb9751
+    type: title
+    task:
+      id: 971290fa-daf4-4510-81d2-610dd2cb9751
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "22"
+      - "40"
+      - "21"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 440,
+          "y": 620
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 59e8a285-2f0f-49c3-83c8-f7126a101b53
+    type: regular
+    task:
+      id: 59e8a285-2f0f-49c3-83c8-f7126a101b53
+      version: -1
+      name: Get filters for the specific forwarding address
+      description: Lists all filters in a user's mailbox.
+      script: '|||gmail-list-filters'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      user-id:
+        simple: ${Core.OriginalAlert.event.raw_log.actor.profileId}
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 830,
+          "y": 280
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: f636844f-1e17-47eb-8b12-e862f2863b85
+    type: regular
+    task:
+      id: f636844f-1e17-47eb-8b12-e862f2863b85
+      version: -1
+      name: Gmail - Get forwarding email address
```

Comment:
Change to "Get forwarding email details" the Gmail -  is unnecessary as this playbook is for google workspace only.